src/test/java/de/retest/web/DefaultValuesProviderTest.java

```diff
@@ -45,4 +45,18 @@ void getDefaultValue_should_prefer_specific_over_general_values() {
 		assertThat( cut.getDefaultValue( "body", "margin-top" ) ).isEqualTo( "8px" );
 		assertThat( cut.getDefaultValue( "boody", "margin-top" ) ).isEqualTo( "0px" );
 	}
+
+	@Test
+	void isDefault_handled_two_null_values() {
```

Comment:
Doesn't the test tests more than "two null values"? It also checks if stuff like `auto`, `normal`, and `0px` is recognized as a default, besides the defaults provided in `defaults.yml`.

Reply:
I'll change it

Reply:
I'll give you a call. 😉